scripts.js

```diff
@@ -141,4 +141,16 @@ function displayProject1(event) {
   </div>`;
 }
 
-popupProjectDetails.forEach((button) => { button.addEventListener('click', (event) => { displayProject1(event); }); });
\ No newline at end of file
+popupProjectDetails.forEach((button) => { button.addEventListener('click', (event) => { displayProject1(event); }); });
+
+const form = document.getElementById('contactme-form');
+const formEmail = document.getElementById('useremail');
+const errorMessage = document.getElementById('error-message');
+
+form.addEventListener('submit', (e) => {
+  if (formEmail.value !== formEmail.value.toLowerCase()) {
```

Comment:
You did a great job making the validation with the proper methodology: `email === email.toLowerCase()` 🔥 Well done!